### ISSUE TITLE
TestDrive is broken on PowerShell Core

### ISSUE THREAD

**Issue by adbertram** [pester/Pester#641]
I'm on MacOS and running a simple test using TestDrive breaks with the following error:

```
[-] Error occurred in Describe block 18ms
   ParameterBindingValidationException: Cannot bind argument to parameter 'Path' because it is null.
   at New-RandomTempDirectory, /usr/local/microsoft/powershell/6.0.0-alpha.12/Modules/Pester/Functions/TestDrive.ps1: line 42
   at New-TestDrive, /usr/local/microsoft/powershell/6.0.0-alpha.12/Modules/Pester/Functions/TestDrive.ps1: line 3
   at Describe, /usr/local/microsoft/powershell/6.0.0-alpha.12/Modules/Pester/Functions/Describe.ps1: line 95
   at <ScriptBlock>, /Users/AdamBertram/Desktop/test.ps1: line 1
   at <ScriptBlock>, /usr/local/microsoft/powershell/6.0.0-alpha.12/Modules/Pester/Pester.psm1: line 451
   at Invoke-Pester, /usr/local/microsoft/powershell/6.0.0-alpha.12/Modules/Pester/Pester.psm1: line 464
   at <ScriptBlock>, <No file>: line 1
Tests completed in 18ms
```

```
PS /Users/AdamBertram> $psversiontable                                                                                                                                                                      

Name                           Value                                                                                                                                                                       
----                           -----                                                                                                                                                                       
PSVersion                      6.0.0-alpha                                                                                                                                                                 
PSEdition                      Core                                                                                                                                                                        
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                     
BuildVersion                   3.0.0.0                                                                                                                                                                     
GitCommitId                    v6.0.0-alpha.12                                                                                                                                                             
CLRVersion                                                                                                                                                                                                 
WSManStackVersion              3.0                                                                                                                                                                         
PSRemotingProtocolVersion      2.3                                                                                                                                                                         
SerializationVersion           1.1.0.1 
```

**Comment by vors** [comment]
I found the same, when trying to evaluate #639

**Comment by adbertram** [comment]
The fix for it is minor and I've submitted a PR. I now need to get past my
Github newbness and wrangle this PR.

**Comment by nohwnd** [comment]
In your PR you have changes that are not in our master because I squashed your commits. You first need to sync your fork's master with the main repo's master. I go to great lenghts about doing that in the [contribution guide](https://github.com/pester/Pester/wiki/Contributing-to-Pester). I am not exactly sure in which state your repo will be after the sync since you've been comitting to you master and not feature branches. (Which is one of the reasons you should be using feature branches.)

**Comment by nohwnd** [comment]
That should have said: I am not sure in what state your repo will be after syncing, so feel free to contact me here, on twitter or skype for further instructions.

**Comment by johlju** [comment]
You could try rebase, then it will sync with master and the commits will be replayed in top. 

```
git checkout master
git fetch origin master
git rebase origin/master
# fix potential merge conflicts
git push my master --force 
```
`--force` on push is important when rebasing. 

`origin` is the remote to Pester/Pester
`my` is the remote to personal fork 
See remotes with `git remote -v`

This is explained more over at PowerShell/DscResources repo in  GettingStartedWithGitHub.md

**Comment by nohwnd** [comment]
Yes rebasing and force pushing is probably a better choice in this case, than the merge mentioned in the official github guide. But to avoid confusion: He is forking the main repo and GitHub automatically set's up the `origin` remote, so his `origin` is not pester/pester, it's adbertram/pester. In your example, the `my` would be `origin` and `origin` would be `upstream`. And he probably does not have the `upstream` set.

So the steps would become:
```
git remote add upstream https://github.com/pester/Pester.git
#review that upstream points to pester/pester and origin to adbertram/pester
git remote -v
git checkout master
git fetch upstream master
git rebase upstream/master
# fix potential merge conflicts

# this will force update your fork's master
git push origin master --force

**Comment by adbertram** [comment]
No worries. I just deleted my fork, started over and submitted a new PR.

**Comment by adbertram** [comment]
This is now under PR #641.

**Comment by nohwnd** [comment]
released in 3.4.5
